pymedphys/_dicom/delivery/core.py

```diff
@@ -125,11 +125,22 @@ def _from_dicom_beam(cls, beam, meterset):
             item.RTBeamLimitingDeviceType for item in beam_limiting_device_sequence
         }
 
-        if rt_beam_limiting_device_types != set(["MLCX", "ASYMY"]):
+        supported_configurations = [{"MLCX", "ASYMY"}]
+
+        if not rt_beam_limiting_device_types in supported_configurations:
             raise ValueError(
                 "Currently only DICOM files that contain "
                 "exactly MLCX and ASYMY are supported. "
```

Comment:
prefer pretty printing the supported_configurations

```
for bld_config in supported_configurations:
    print(" and ".join(bld_config))
```

or something equivalent

Reply:
Good call. DRY > WET :)